internal/goreleaser/goreleaser.go

```diff
@@ -61,6 +61,11 @@ func RenderConfig(cfg core.Configuration) {
 		}
 	}
 
+	branch := "master"
+	if cfg.GitHubWorkflow != nil && cfg.GitHubWorkflow.Global.DefaultBranch != "" {
```

Comment:
Not to sure if we want to do this or introduce another config field. I'm open for suggestions.